lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -271,14 +271,14 @@ function _campaign_resource_retrieve($nid) {
 /**
  * Retrieve and show index list response of campaigns requested.
  *
- * @param string $ids
- * @param bool $staff_pick
- * @param bool $mobile_app
- * @param string $mobile_app_date
- * @param string $term_ids
- * @param int $count
- * @param bool $random
- * @param int $page
+ * @param  string  $ids
```

Comment:
@uy yeah, I was concerned about that too, but curious to see what you think of how Taylor handles that same issue in Laravel. See [here](https://github.com/laravel/framework/blob/1e55c79fd1d3e778d867a43f5495957fcb52a655/src/Illuminate/Cache/DatabaseStore.php#L45) for an example.